apps/web/client/src/app/_components/hero/index.tsx

```
@@ -15,6 +16,21 @@ export function Hero() {
         <div className="w-full h-full flex flex-col items-center justify-center gap-12 p-8 text-lg text-center relative">
             <UnicornBackground />
             <div className="flex flex-col gap-3 items-center relative z-20 pt-4 pb-2">
+                <motion.div 
+                    className="flex flex-col gap-3 items-center relative z-20 pt-4 pb-2 mb-6"
+                    initial={{ opacity: 0, y: -10 }}
+                    animate={{ opacity: 1, y: 0 }}
+                    transition={{ duration: 0.6, delay: 1.2, ease: "easeOut" }}
+                >
+                    <a 
+                        href="https://www.ycombinator.com/companies/onlook/jobs/e4gHv1n-founding-engineer-fullstack" target="_blank" 
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)